crowdfund/contracts/SmartPoolToken.sol

```diff
@@ -0,0 +1,131 @@
+pragma solidity ^0.4.0;
+
+import "./ERC20.sol";
+import "./Lockable.sol";
+
```

Comment:
I think @tranvictor follows the ERC20 template from https://github.com/ethereum/EIPs/issues/20